luda-editor/server/server-core/src/services/sequence/mod.rs

```diff
@@ -290,15 +293,33 @@ impl rpc::SequenceService<SessionDocument> for SequenceService {
                 return Err(rpc::delete_sequence::Error::Unauthorized);
             }
 
-            crate::dynamo_db()
+            let memo_ids = MemoDocumentQuery {
+                pk_sequence_id: req.sequence_id,
+            }
+            .run()
+            .await
+            .map_err(|error| rpc::delete_sequence::Error::Unknown(error.to_string()))?
+            .into_iter()
+            .map(|memo| memo.memo_id);
+
+            let transaction = crate::dynamo_db()
                 .transact()
                 .delete_item(SequenceDocumentDelete {
                     pk_id: req.sequence_id,
                 })
                 .delete_item(ProjectSequenceDocumentDelete {
                     pk_project_id: sequence.project_id,
                     sk_sequence_id: req.sequence_id,
+                });
+
+            let transaction = memo_ids.fold(transaction, |transaction, memo_id| {
+                transaction.delete_item(MemoDocumentDelete {
```

Comment:
transaction only handle 100 actions, so it would be problem.

Reply:
I think you can delete without transaction, just delete memos first and if no fail, delete other things in transaction.

Reply:
I think so too. Fixed with 7550e4a